packages/graphiql/cypress/e2e/tabs.cy.ts

```diff
@@ -1,5 +1,23 @@
 describe('Tabs', () => {
   it('Should store editor contents when switching between tabs', () => {
```

Comment:
```suggestion
  describe('confirmCloseTab()', () => {
    it('should keep tab when `Cancel` was clicked', () => {
      cy.on('window:confirm', () => false);
      cy.visit('/');

      cy.get('.graphiql-tab-add').click();

      cy.get('.graphiql-tab-button + .graphiql-tab-close').eq(1).click();

      cy.get('.graphiql-tab-button').should('have.length', 2);
    });

    it('should close tab when `OK` was clicked', () => {
      cy.on('window:confirm', () => true);
      cy.visit('/');

      cy.get('.graphiql-tab-add').click();

      cy.get('.graphiql-tab-button + .graphiql-tab-close').eq(1).click();

      cy.get('.graphiql-tab-button').should('have.length', 1);
    });
  })

  it('Should store editor contents when switching between tabs', () => {
```

add this test instead, it's much clear to understand